tests/python/frontend/tflite/test_forward.py

```diff
@@ -143,11 +144,13 @@ def compare_tflite_with_tvm(in_data, in_name, input_tensors,
             converter.inference_type = tf.lite.constants.QUANTIZED_UINT8
             input_arrays = converter.get_input_arrays()
             input_stats = {}
-            # hardcode the mean_values and std_dev_values (m,s) to be the same
-            # if all inputs are in (float_min; float_max) == (-100, 100)
+            # calculate the mean and quantization scale for every input tensor,
+            # with respect to its fp32 input range, defined in fake_quant.
             # s = 255/(fmax-fmin);  m = -fmin*s (the zero point)
             for i in input_arrays:
-                input_stats[i] = (128., 1.275)
+                quant_scale = 255 / (input_range[i][1] - input_range[i][0])
```

Comment:
Minor
* Lets add a check to ensure that denominator is not zero.
* Typically QNN_scale = FP32_range/QNN_range, so suggest to double-check if TFLite accepts inverse scale.

Reply:
Thanks for reviewing! Adding a check makes sense - I will do it. 
Here you can see the documentation I used for developing the qnn tests
- at 'Quantizing models for integer-only execution':  **_std_dev is  255 / (float_max - float_min)_**  https://www.tensorflow.org/lite/convert/quantization
- at 'Model flags': **_real_value = (quantized_input_value - mean_value) / std_dev_value_**
https://www.tensorflow.org/lite/convert/cmdline_reference